index.html

```diff
@@ -540,8 +540,8 @@ <h4>F-Droid</h4>
 
                                             </div>
                                             <div class="item">
-                                                <p>F-Droid also guarantees the best possible privacy and therefore does not track its users. <br />
-                                                    At this point we would like to thank F-Droid for providing the downloads.
+                                                <p>F-Droid also guarantees the best possible privacy and does not track its users.<br />
```

Comment:
F-Droid don't guarantee anything, they try their best and do a great job IMO. Let's be a bit more realistic in this sentence and not make any promises.

Reply:
I was thinking about it, but didnt dare to change that much on my own directly.